packages/pigment-css-react/src/baseAtomics.js

```diff
@@ -0,0 +1,43 @@
+import { generateAtomics } from './generateAtomics';
+
+export const stackAtomics = generateAtomics(({ theme }) => ({
+  conditions: Object.keys(theme.breakpoints.values).reduce((acc, breakpoint) => {
+    acc[breakpoint] = `@media (min-width: ${theme.breakpoints.values[breakpoint]}${
+      theme.breakpoints.unit ?? 'px'
+    })`;
+    return acc;
+  }, {}),
+  defaultCondition: theme.breakpoints?.keys?.[0] ?? 'xs',
+  properties: {
+    display: ['flex', 'inline-flex'],
+    flexDirection: ['column', 'column-reverse', 'row', 'row-reverse'],
+    justifyContent: [
+      'end',
+      'start',
+      'flex-end',
+      'flex-start',
+      'center',
+      'space-between',
+      'space-around',
+      'space-evenly',
+    ],
+    alignItems: [
+      'center',
+      'end',
+      'flex-end',
+      'flex-start',
+      'self-end',
+      'self-start',
+      'start',
+      'baseline',
+      'normal',
+      'stretch',
+    ],
+    gap: ['--Stack-gap'],
+  },
+  shorthands: {
+    direction: ['flexDirection'],
+    spacing: ['gap'],
+  },
+  multiplier: 8,
```

Comment:
Did you figure out how to take this from the theme?

Reply:
@siriwatknp What's your opinion here? `theme.spacing()` without arguments does not guarantee a numeric value.

Reply:
I think it should always be a string, then use CSS `calc()` as a value. To leverage the theme, use 

```js
multiplier: theme.vars.spacing, // it can be a string or an array.
```